emmet-core/emmet/core/vasp/material.py

```diff
@@ -54,6 +54,12 @@ def from_tasks(
             quality_scores: quality scores for various calculation types
             use_statics: Use statics to define a material
         """
+        if task_group == 0:
```

Comment:
Should there be `len` in there?

Reply:
Yeah I'll add a test for and update this PR